bin/build-tag-push-docker-image.sh

```diff
@@ -1,3 +1,39 @@
 #!/usr/bin/env bash
+#TODO write a script that will push all our docker images to GC so we can use them
+#Before you start be sure, THAT billing is enabled for your Google Cloud project and Enable the Container Registry API.
+#Install and initialize the Cloud SDK using this link https://cloud.google.com/sdk/docs/?hl=de and install docker with this link https://docs.docker.com/install/linux/docker-ce/ubuntu/
 
-#TODO write a script that will push all our docker images to GC so we can use them
\ No newline at end of file
+
+
+#Build an image from a Dockerfile-Xhorxhina
+
+cd ~/cp_ws_1920/Docker
```

Comment:
this is still an absolute path. please take a moment to understand what is the difference between a relative path and an absolute path